### PR TITLE
[ANCHOR-680] Create dependabot.yml

### PR DESCRIPTION
**Pull request by JiahuiWho**
### Description

Create dependabot to auto update dependency

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      interval: "weekly"
+      time: "09:00"
+      timezone: "US/Pacific"
+    commit-message:
+      prefix: "[Gradle]"
+    open-pull-requests-limit: 10
+    target-branch: develop
+    labels:
+      - "gradle"
+      - "vulnerability"
```
Comment (lijamie98) [inline .github/dependabot.yml:20]: Did you create the label `vulnerability` and `gradle` in the repo?

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      - "dependencies"
+
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      time: "09:00"
+      timezone: "US/Pacific"
+    commit-message:
+      prefix: "[Docker]"
```
Comment (lijamie98) [inline .github/dependabot.yml:30]: Love this one.

**Review by lijamie98 — APPROVED** [review]
LGTM. Please feel free to merge after the comments are taken care of.

File: .github/dependabot.yml (hunk 1 above)
Comment (JiahuiWho) [reply .github/dependabot.yml:20]: Yes :)

**Pull request merged**